src/locales/en.json

```diff
@@ -41,6 +42,8 @@
   "header_team": "Team",
   "header_toggle": "Toggle navigation",
   "header_values": "Our Values",
+  "impressum_title": "Legal Notice",
+  "impressum_text": "<p class=\"mb-3\">Die Lernwerkstatt - lernen und lernen lassen e.V.<br />Karl-Kunger-Str. 55<br />12435 Berlin <br /><a href=\"mailto:info@die-lernwerkstatt.org\" target=\"_blank\" rel=\"noopener\">info@die-lernwerkstatt.org</a><br /><a href=\"tel:+49 176 87840154\">+49 176 87840154</a></p><p class=\"mb-3\">Vertretungsberechtigter Vorstand des Vereins: André Brötz & Bianca Geburek<br />Amtsgericht Charlottenburg<br />VR 34225 B<br />Steuernr. : 27/658/52900</p>",
```

Comment:
Legal Notice is an `Imprint`, as far as I know :)
Could please also translate the rest of the text? Like `Vertretungsberechtigter Vorstand des Vereins`, for example?